content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -126,10 +129,10 @@ template:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.labels['tags.datadoghq.com/service']
-          # - name: DD_VERSION #Uncomment this line to manually tag your version
-          #   valueFrom: #Uncomment this line to manually tag your version
-          #     fieldRef: #Uncomment this line to manually tag your version
-          #       fieldPath: metadata.labels['tags.datadoghq.com/version'] #Uncomment this line to manually tag your version
+          - name: DD_VERSION #Comment out this line to use automatic version tagging in APM
```

Comment:
I think this is the only thing that you'd want to comment out.

From the labels alone we should be able to put version on logs/metrics.

It's technically only the APM tracer that needs the `DD_VERSION` env var in order to set version itself.

Reply:
Removed all messaging to comment this out, to reduce confusion until we have this feature enabled for all product data